website/docs/reference/integrations/jira-cloud-plugin-usage.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Jira Cloud Integration - Usage
+---
+
+With the Unleash Jira Cloud Plugin you can create, view and manage Unleash feature toggles directly from a Jira issue.
+
+The plugin also shows you current status of connected toggles.
+
+## Using the plugin
+
+To use the plugin, you'll need to create an issue or use an existing one. Once the issue is saved and you open up the issue panel, you'll be greeted with a button to activate the Unleash plugin for that issue.
+
+![Jira Cloud - Button to activate Unleash plugin in Jira Issue](/img/jira-cloud-activate-unleash-panel-button.png)
```

Comment:
What do you call that row of buttons in Jira? I suggest something like this, but I don't know what to call that row.

Anyway, maybe something like this?

```suggestion
<Figure caption="The Unleash Feature Flags button gets added to the top-level actions of new issues. Use that button to activate the Unleash plugin for that issue." img="/img/jira-cloud-activate-unleash-panel-button.png" />
```

Reply:
The section the plugin content ends up in is called a Jira Issue Panel. I can't find a name mentioned anywhere for that row however. Maybe just go with your suggestion here